proton-j/src/main/java/org/apache/qpid/proton/engine/impl/BuiltinEventTypeImpl.java

```diff
@@ -0,0 +1,144 @@
+package org.apache.qpid.proton.engine.impl;
+
+import org.apache.qpid.proton.engine.BuiltinHandler;
+import org.apache.qpid.proton.engine.Event;
+import org.apache.qpid.proton.engine.Handler;
+
+public class BuiltinEventTypeImpl {
```

Comment:
This is out-of-line implementation of EventType interface for Event.Type#dispatch.

CoreEventDispatcher?